### ISSUE TITLE
loosing items out of the golden bag of holding

### ISSUE THREAD

**Issue by DeathRaye** [rwtema/extrautilities_old_issues#1366]
Using version 1.10.2-1.3.0

**Comment by mcdodge34** [comment]
I had the issue today, and I suspect its linked to the fact that I switched in Creative mode for testing purposes and I moved the bag of holding out of my hotbar to my inventory from the creative mode, maybe its linked, but I use version 1.10.2-1.3.2

**Comment by xenxero1** [comment]
Any time you are op-ed, de-oped or enter/leave creative it seems to empty.

**Comment by mcdodge34** [comment]
Thanks for confirming this, now I know I'll leave my bags in a chest next time, now if I could find a way to see what I had in my bag previously, I have backups but not sure if I can see what was in my bag before, where its stored exactly ???

**Comment by xenxero1** [comment]
I don't believe that data is saved anywhere, or else we'd already have an easy fix. Sadly you'l just have to try and remember a bulk of the items.

**Comment by mcdodge34** [comment]
I'll restore a backup in a single player world and see what I had in this bag, it wasn't very important stuff cause I hate the fact we can't use Inventory Tweaks on it so I avoid using it, but I had still some stuff in there like enderio item conduits that aren't easy to craft while just starting out since they require ender pearls.

**Comment by savosdarkran** [comment]
I had this issue a few moments ago in 1.11.2.  It seems to have followed into the new version.  Any word on a fix?  Or is this just something we will need to remember?  Either way, please let me know.

Cheers.

**Comment by ExDomino** [comment]
Items are lost after you pass from creative mode to survival mode, even in 1.12 with the latest version.
Do not use the bag of holding if you are admin!

**Comment by mcdodge34** [comment]
Good to know, I prefer Iron Backpacks (still waiting on a 1.12 version of this actually) generally, or RFTools tablet which can hold 300 stacks so much bigger than many backpack mods.

**Comment by ResqDiver1317** [comment]
We have been dealing with this on our server as well (new custom built 1.10.2 modpack. We narrowed it down to (at least for us) moving the slot the bag is in while in creative mode will empty it. If we don't touch the bag while in creative, it's fine, but if we touch it at all, gone... We have started just throwing our bags in our Enderchests before doing any creative work on the server.

**Comment by Sukasa** [comment]
This just bit me in 1.12.2.

**Comment by Teslachu** [comment]
I also just had this happen on a server, in 1.12.2... Though I went into creative, only to go into flight for a second and check something out in twilight forest. I keep the bag on my bar. Checked what I needed to, returned to survival... and decided to open my bag for some of my mats and boom - everything was gone.
